.idea/codeStyles/Project.xml

```diff
@@ -179,6 +179,8 @@
       <option name="FOR_BRACE_FORCE" value="3" />
     </codeStyleSettings>
     <codeStyleSettings language="XML">
+      <option name="LINE_COMMENT_AT_FIRST_COLUMN" value="false" />
```

Comment:
What is the effect of these two settings?

Reply:
When you press the keyboard shortcut to comment stuff out, do the comment markers go at the current indent level or in column 0? For some reason the default is column 0 in both Java and XML. We already reconfigured it for Java years ago, but for some reason never configured it for XML. I noticed while working on the maven profiles.